test/wiki-report.py

```diff
@@ -117,4 +117,11 @@ def run(self):
         print("Usage: compose-report.py <report_path>")
         sys.exit(1)
 
+    try:
+        with open("~/.openidc/oidc_wikitcms.json", "r") as f:
```

Comment:
home dir is impractical for reasons -- please check `os.getenv("COCKPIT_FEDORA_WIKI_TOKEN")` and if it's set, then that is the path of the file.

Reply:
Oh I see -- if it is set, please copy it to that destination path. We can't mount it as volume from outside with the correct permissions.

Reply:
See https://github.com/cockpit-project/cockpituous/pull/642